src/managers/legal-hold-policies.ts

```diff
@@ -0,0 +1,391 @@
+	 *
+	 * @param {string} name - The name of the legal hold policy to be created
+	 * @param {Object} [options] - Additional parameters
+	 * @param {string} [options.description] - Description of the legal hold policy
+	 * @param {string} [options.filter_started_at] - Date filter, any Custodian assignments will apply only to file versions created or uploaded inside of the date range
+	 * @param {string} [options.filter_ended_at] - Date filter, any Custodian assignments will apply only to file versions created or uploaded inside of the date range
+	 * @param {boolean} [options.is_ongoing] - After initialization, Assignments under this Policy will continue applying to files based on events, indefinitely
+	 * @param {Function} [callback] - Passed the new policy information if it was acquired successfully, error otherwise
+	 * @returns {Promise<Object>} A promise resolving to the created policy
+	 */
+	create(
+		name: string,
+		options?: {
+			description?: string;
+			filter_started_at?: string;
+			filter_ended_at?: string;
+			is_ongoing?: boolean;
+		},
+		callback?: Function
+	) {
+		var apiPath = urlPath(BASE_PATH),
+			params: Record<string, any> = {
+				body: options || {},
+			};
+
+		params.body.policy_name = name;
+
+		return this.client.wrapWithDefaultHandler(this.client.post)(
+			apiPath,
+			params,
+			callback
+		);
+	}
+
+	/**
+	 * Fetches details about a specific legal hold policy
+	 *
+	 * API Endpoint: '/legal_hold_policies/:policyID'
+	 * Method: GET
+	 *
+	 * @param {string} policyID - The Box ID of the legal hold policy being requested
+	 * @param {Object} [options] - Additional options for the request. Can be left null in most cases.
+	 * @param {Function} [callback] - Passed the policy information if it was acquired successfully, error otherwise
+	 * @returns {Promise<Object>} A promise resolving to the policy object
+	 */
+	get(policyID: string, options?: Record<string, any>, callback?: Function) {
+		var apiPath = urlPath(BASE_PATH, policyID),
+			params = {
+				qs: options,
+			};
+
+		return this.client.wrapWithDefaultHandler(this.client.get)(
+			apiPath,
+			params,
+			callback
+		);
+	}
+
+	/**
+	 * Update or modify a legal hold policy.
+	 *
+	 * API Endpoint: '/legal_hold_policies/:policyID'
+	 * Method: PUT
+	 *
+	 * @param {string} policyID - The Box ID of the legal hold policy to update
+	 * @param {Object} updates - The information to be updated
+	 * @param {string} [updates.policy_name] - Name of Legal Hold Policy
+	 * @param {string} [updates.description] - Description of Legal Hold Policy
+	 * @param {string} [updates.release_notes] - Notes around why the policy was released
+	 * @param {Function} [callback] - Passed the updated policy information if it was acquired successfully, error otherwise
+	 * @returns {Promise<Object>} A promise resolving to the updated policy
+	 */
+	update(
+		policyID: string,
+		updates: {
+			policy_name: string;
+			description: string;
+			release_notes: string;
```

Comment:
based on the API, these should probably be optional types https://developer.box.com/reference/put-legal-hold-policies-id/#body-parameters

Reply:
right, thanks! :)